lib/tasks/report.rake

```diff
@@ -73,21 +73,22 @@ namespace :app do
     # Part of: https://www.pivotaltracker.com/story/show/165217423
     # The idea is to update about ~500K learner-runs in LARA to include
     # `class_hash` values, which we need for FireStore authorization Rules
-    # LINE FORMAT: CLAZZ_ID, CLASS_HASH, LEARNER_ID, LEARNER_KEY
+    # LINE FORMAT: CLAZZ_ID, CLASS_HASH, USER_ID, LEARNER_KEY
```

Comment:
```suggestion
    # LINE FORMAT: CLAZZ_ID, CLASS_HASH, LEARNER_ID, LEARNER_KEY, USER_ID, OFFERING_ID
```
We still need the learner id to construct the old remote_endpoints that were used before the learner_key was introduced.